polls/tool/src/webapp/templates/votePolls.html

```diff
@@ -108,9 +109,9 @@ <h1 rsf:id="poll-list-title">Polls</h1>
       </td>
     </tr>
   </tbody>
-</table>
+</table></div>
 
-<p class="act">
+<p class="act" style="flex-wrap:nowrap;">

```

Comment:
prefer not using inline style

Reply:
Thanks! I updated the code with a bootstrap class instead.